05-history.md

```diff
@@ -6,10 +6,60 @@ minutes: 25
 ---
 > ## Learning Objectives {.objectives}
 >
+> *   Learn what the HEAD of a repository is and how it's useful
 > *   Identify and use Git commit numbers.
 > *   Compare various versions of tracked files.
 > *   Restore old versions of files.
 
+As we saw in the previous lesson, we can refer to commits by their
+identifiers, but even the short ones do not spring trippingly from the tongue.
+Much of the time, we'll be working on some revisions, and when we want to
+refer back to something in a previous commit, it might be nice not to have to
+look up the identifier.  The clever Git people thought so, too, so you can
+refer to the _most recent commit_ by using `HEAD`.
```

Comment:
I would not merge this. This is, as @QuLogic says, not accurate. HEAD is a pointer to the branch you are in. If you are in something that isn't a branch, you are in "detached HEAD" mode. Introducing HEAD at this point is not necessary, and might confuse students more than anything else. I think that should be moved to after introducing branches.